src/client/sandbox/event/drag-and-drop/data-transfer-item-list.js

```diff
@@ -13,9 +13,9 @@ function parseTextUriList (textUriList) {
 
     textUriList = textUriList.split(/\r\n/);
 
-    for (let i = 0; i < textUriList.length; i++) {
-        if (textUriList[0] !== '#')
```

Comment:
I propose to apply this small fix.
Also, I've created the separate issue for uri-list conversion #1248.

Note, I've checked uri-list conversion in IE, Firefox and Chrome and it work nowhere.
There is the bug for this case - https://bugs.chromium.org/p/chromium/issues/detail?id=239745